<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1554</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PS Acquisition Tool</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="mplwindow" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>601</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>9</y>
       <width>191</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Acquisition</string>
     </property>
     <widget class="QPushButton" name="btnCamera">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>70</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Acquire image from camera and display results</string>
      </property>
      <property name="text">
       <string>Capture</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFromDir">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>110</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Process images from archive and display results</string>
      </property>
      <property name="text">
       <string>From Archive</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="gbMode">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>20</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string>Mode</string>
      </property>
      <widget class="QRadioButton" name="rbVIS">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>17</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>VIS</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbNIR">
       <property name="geometry">
        <rect>
         <x>71</x>
         <y>17</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>NIR</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbVISNIR">
       <property name="geometry">
        <rect>
         <x>127</x>
         <y>16</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Both</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="grpConfigProperties">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>191</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>Processing options</string>
     </property>
     <widget class="QCheckBox" name="chkBoxLightingComp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Compensates via inverse square law</string>
      </property>
      <property name="text">
       <string>Lighting compensation</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkBoxCrop2Plant">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Crops the reconstruction to the plant only (based on albedo thresholding)</string>
      </property>
      <property name="text">
       <string>Crop to plant only</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkBoxSubtractAmbient">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Crops the reconstruction to the plant only (based on albedo thresholding)</string>
      </property>
      <property name="text">
       <string>Subtract ambient</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRecalc">
      <property name="geometry">
       <rect>
        <x>104</x>
        <y>230</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Recalculate the normals etc using the selected settings</string>
      </property>
      <property name="text">
       <string>Re-process</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExport">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Archive the results of the current settings</string>
      </property>
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>161</width>
        <height>100</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Specify Manually or Draw a region in Nx</string>
      </property>
      <property name="title">
       <string>Region of Interest</string>
      </property>
      <widget class="QLineEdit" name="edtX1">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="edtX2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="edtY2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="edtY1">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>30</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>x1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>95</x>
         <y>30</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>x2</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>60</y>
         <width>20</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>y1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>95</x>
         <y>60</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>y2</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="mplwindow2" native="true">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>10</y>
      <width>661</width>
      <height>661</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="mplvl2"/>
   </widget>
   <widget class="QWidget" name="mplwindow3" native="true">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>661</width>
      <height>661</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="mplvl3"/>
   </widget>
   <widget class="QProgressBar" name="pbProgress">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>670</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblProgress">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>674</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>!Status</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>rbVIS</tabstop>
  <tabstop>rbNIR</tabstop>
  <tabstop>rbVISNIR</tabstop>
  <tabstop>btnCamera</tabstop>
  <tabstop>btnFromDir</tabstop>
  <tabstop>chkBoxLightingComp</tabstop>
  <tabstop>chkBoxCrop2Plant</tabstop>
  <tabstop>chkBoxSubtractAmbient</tabstop>
  <tabstop>edtX1</tabstop>
  <tabstop>edtX2</tabstop>
  <tabstop>edtY1</tabstop>
  <tabstop>edtY2</tabstop>
  <tabstop>btnExport</tabstop>
  <tabstop>btnRecalc</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
